mycroft/util/__init__.py

```diff
@@ -364,3 +366,11 @@ def wait_while_speaking():
     time.sleep(0.1)  # Wait briefly in for any queued speech to begin
     while is_speaking():
         time.sleep(0.1)
+
+
+def stop_speaking():
+    # TODO: Less hacky approach to this once Audio Manager is implemented
+    # Skills should only be able to stop speech they've initiated
+    config = mycroft.configuration.ConfigurationManager.instance()
+    kill([config.get('tts').get('module')])
+    kill(["aplay"])
```

Comment:
Won't this leave the visimes running on the display?

Couldn't this check `is_speaking()` and if so `create_signal('buttonPress')` to stop speech.